index.html

```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
   </head>
   <body>
-    <h1>Hello from Rust!</h1>
+    <h1>Hello Preston</h1>
```

Comment:
Exclamation mark to show excitement
```suggestion
    <h1>Hello Preston!</h1>
```